python/rikai/types/geometry.py

```diff
@@ -263,22 +263,40 @@ def area(self) -> float:
         """Area of the bounding box"""
         return self.width * self.height
 
+    @staticmethod
+    def _area(arr: np.ndarray) -> np.array:
+        """Calculate box area from an array of boxes."""
+        return np.maximum(0, arr[:, 2] - arr[:, 0]) * np.maximum(
+            0, arr[:, 3] - arr[:, 1]
+        )
+
     def iou(self, other: "Box2d") -> float:
         """Compute intersection over union(IOU)."""
         assert isinstance(
-            other, Box2d
+            other, (Box2d, list)
```

Comment:
what about like a tuple or array of Box2d?

Reply:
done